src/zenml/zen_stores/sql_zen_store.py

```diff
@@ -784,12 +787,10 @@ def alembic(self) -> Alembic:
     def filter_and_paginate(
         cls,
         session: Session,
-        query: Union[Select[AnySchema], SelectOfScalar[AnySchema]],
+        query: Union[Select[Any], SelectOfScalar[Any]],
```

Comment:
I understand this now.

Reply:
My current query does not query the entire `PipelineSchema` but just the `name`. This is so that the `distinct()` method works which gives me the names without duplicates but still filtered/paginated correctly.